### PR TITLE
[ADD] [13.0] stock_inventory_include_exhausted:New module to create inventory lines including exhausted products

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT22120

### CODE REVIEW

File: stock_inventory_include_exhausted/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* Carlos Roca (Tecnativa) <carlos.roca@tecnativa.com>
```
Comment (manuelcalerosolis) [inline stock_inventory_include_exhausted/readme/CONTRIBUTORS.rst]: * `Tecnativa <https://www.tecnativa.com>`_:

  * Carlos Roca

**Review by manuelcalerosolis — APPROVED** [review]
Code and functional review

File: stock_inventory_include_exhausted/tests/test_stock_inventory_include_exhausted.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class StockInventoryIncludeExhaustedTest(TransactionCase):
+    def setUp(self):
+        super(StockInventoryIncludeExhaustedTest, self).setUp()
```
Comment (Tardo) [inline stock_inventory_include_exhausted/tests/test_stock_inventory_include_exhausted.py]:
```suggestion
        super().setUp()
```

**Review by Tardo — APPROVED** [review]
:+1: LGTM, tested on runbot. Only a non-blocking comment...

**Comment by CarlosRoca13** [comment]
@manuelcalerosolis @Tardo Changes done!!

File: stock_inventory_include_exhausted/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+{
+    "name": "Inventory Include Exhausted",
+    "summary": "It includes the option for adding products exhausted on the "
+    "inventories.",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "depends": ["stock"],
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
```
Comment (rousseldenis) [inline stock_inventory_include_exhausted/__manifest__.py]:
```suggestion
    "category": "Warehouse",
```

**Review by rousseldenis — APPROVED** [review]
Code review. Minor remark

**Comment by rousseldenis** [comment]
Don't forget pre-commit

**Comment by CarlosRoca13** [comment]
@rousseldenis change done!

**Comment by rousseldenis** [comment]
So, this is ready?

**Comment by CarlosRoca13** [comment]
@rousseldenis I think

**Pull request merged**